fhir-persistence/src/main/java/com/ibm/fhir/persistence/util/FHIRPersistenceUtil.java

```diff
@@ -128,11 +141,17 @@ public static FHIRSystemHistoryContext parseSystemHistoryParameters(Map<String,
                                         .withIssue(FHIRUtil.buildOperationOutcomeIssue(msg, IssueType.INVALID));
                             }
                             // Note: if we decide to support invalid _type values in 'lenient' mode (like search), then the following
-                            // if/else will need to be in an else block for the preceding if
+                            // if/else block will need to be in an else block for the preceding if
                             if (!isHistoryEnabled(resourceType)) {
                                 String msg = "history interaction is not supported for _type parameter value: " + Encode.forHtml(resourceType);
                                 throw new FHIRPersistenceException(msg)
                                         .withIssue(FHIRUtil.buildOperationOutcomeIssue(msg, IssueType.NOT_SUPPORTED));
+                            } else if (fhirVersion == FHIRVersionParam.VERSION_40 &&
+                                    ResourceTypeHelper.getNewOrBreakingResourceTypeNames().contains(resourceType)) {
```

Comment:
minor, but this test could be made a method of ResourceTypeHelper which allows you to hide the implementation.

Reply:
I like this suggestion a lot.  I think checks like this are the only use of `getNewOrBreakingResourceTypeNames` so I can probably remove that if I introduce the new type.  will see how it works out

Reply:
at first I was thinking to have the utility just return whether or not a given resource type is supported for a given FHIR Version.
the "problem" with this is that a couple of the resource types *were* in R4, but have breaking changes in R4B (e.g. Evidence).  it would be wrong to say that Evidence "isn't supported" in R4 in general.
its really a *fhir-server* decision to say "we don't support version 4.0 of Evidence"

Reply:
the changeset was getting too big, so I decided to split this into a separate branch/PR for discussion